packages/mdc-checkbox/_mixins.scss

```diff
@@ -186,6 +186,7 @@
     @include mdc-ripple-surface($query);
     @include mdc-states($mdc-checkbox-baseline-theme-color, $query: $query);
     @include mdc-ripple-radius-unbounded($query: $query);
+    @include mdc-checkbox-touch-dimension($mdc-checkbox-touch-area);
```

Comment:
I need to double check this, but I think this should be outside of .mdc-checkbox